ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -552,7 +570,7 @@ data TraceAddBlockEvent blk
     -- as the new chain.
 
   | SwitchedToAFork
-      (RealPoint blk)
+      (AdoptedBlockInfo blk)
```

Comment:
Switch comment style, explain that the `RealPoint` here also isn't necessarily the new tip.